assets/sass/_mixins.scss

```diff
@@ -398,22 +404,19 @@
 }
 
 @mixin popup_title_text {
-  @include font-size(14);
-  line-height: 1.4285714286; /* 20px on 24px font */
+  @include font-size-and-vertical-height(14, 20);
```

Comment:
Are we able to parameterise most/all of these values? They're a bit magic otherwise.

Reply:
A lot have been, but more should be able to be, yes.